build/script/UpdateManifestJsonForSbom.ps1

```diff
@@ -1,29 +0,0 @@
-# Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
```

Comment:
This workaround was no longer needed since we generate the SBOM element via SwixBuild now.